### DISCUSSION TITLE
Self-nomination to Contributor position

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#154] [Stacks Team]
Hi! I'd like to put myself forward as a candidate to join the Contributor team. I've worked a bit on refactoring the codebase, and some SBOM-related items. You can see those commits [here](https://github.com/paketo-buildpacks/stacks/commits?author=ryanmoran)

@paketo-buildpacks/stacks-maintainers can I get a 👍 / 👎 ?

**Comment by sophiewigmore** [comment]
@ryanmoran we are at 2/2 approvals on this!

**Comment by ryanmoran** [comment]
🎉
